plugins/outputs/timestream/timestream.go

```diff
@@ -279,7 +279,7 @@ func (t *Timestream) writeToTimestream(writeRecordsInput *timestreamwrite.WriteR
 		var notFound *types.ResourceNotFoundException
 		if errors.As(err, &notFound) {
 			if resourceNotFoundRetry {
-				t.Log.Warnf("Failed to write to Timestream database '%s' table '%s'. Error: '%s'",
+				t.Log.Warnf("Failed to write to Timestream database %q table %q: %w",
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.